website/docs/d/datastore.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "vsphere"
+page_title: "VMware vSphere: vsphere_datastore"
+sidebar_current: "docs-vsphere-data-source-datastore"
+description: |-
+  Provides a vSphere datastore data source. This can be used to get the general attributes of a vSphere datastore.
+---
+
+# vsphere\_datastore
```

Comment:
_minor_ you don't actually need the backslash here

Reply:
Yeah, this seems to be in a _lot_of places ([example in the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/website/docs/r/api_gateway_client_certificate.html.markdown)). Probably needs to be part of a larger cleanup effort as it is also in all of our docs here too.